contrib/mixin/mixin.libsonnet

```diff
@@ -223,7 +223,7 @@
           {
             alert: 'etcdExcessiveDatabaseGrowth',
             expr: |||
-              increase(((etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100)[240m:1m]) > 50
+              deriv(((etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100)[240m:1m]) > 50
```

Comment:
Pretty sure this will not have the same threshold value after using deriv here, curious to know how did you test this? 🤔

Reply:
Hi @lilic, I haven't tested this, my bad. I will test this and update it as required. Thanks.

Reply:
@chaitanyaenr did you manage to have a look and test it? Thanks!